src/Integration/Symfony/Bundle/composer.json

```diff
@@ -23,9 +23,14 @@
         "async-aws/ses": "^1.0",
         "async-aws/sqs": "^1.0 || ^2.0",
         "async-aws/ssm": "^1.0 || ^2.0",
-        "matthiasnoback/symfony-config-test": "^4.1 || ^5.0",
+        "matthiasnoback/symfony-config-test": "^6.1",
         "nyholm/symfony-bundle-test": "^3.0",
-        "symfony/cache": "^4.4 || ^5.0 || ^6.0 || ^7.0 || ^8.0"
+        "phpunit/phpunit": "^11.5.42",
+        "symfony/cache": "^4.4 || ^5.0 || ^6.0 || ^7.0 || ^8.0",
+        "symfony/error-handler": "^7.3.2 || ^8.0",
+        "symfony/framework-bundle": "^5.4.45 || ^6.4.13 || ^7.1.6",
```

Comment:
This requirement on FrameworkBundle is meant to fix our `--prefer-lowest` CI job by restrict the lowest bound (for the dependency coming from `nyholm/symfony-bundle-test`) so that we have a version including the change in https://github.com/symfony/framework-bundle/commit/1c11e27c68c3dd576edf93dcb7a36087b80a50df

this change combined with the fact that I install `symfony/runtime` solves the issue that booting the bundle would attempt at replacing the error handler, causing issues with PHPUnit 11 (this is why I don't let the composer plugin run for the runtime component: I won't care about it).